configuration-sample/ods-core.env.sample

```diff
@@ -369,3 +369,24 @@ ADDITIONAL_JIRA_PROJECT_TEMPLATE_PROPERTIES="# ---------------------------------
 
 # All params defined in the template https://github.com/opendevstack/ods-core/blob/master/ods-provisioning-app/ocp-config/dc.yml
 # can be overwritten by specifying the param in this file.
+
+########
+# Aqua #
+########
+# Enable or disable Aqua at the cluster level
+AQUA_ENABLED=false
+
+# The URL to the Aqua service
+AQUA_URL=
+
+# The name of a container image registry in Aqua
+AQUA_REGISTRY=
+
+# The name of a secret containing credentials (username and password) of an Aqua user that has scanner rights
+AQUA_SECRET_NAME=
+
+# List of email addresses to provide feedback in case there are problems using Aqua
+AQUA_ALERT_EMAILS=
+
+# The name of a Nexus repository to store the reports generated by Aqua
+AQUA_NEXUS_REPOSITORY=
```

Comment:
Should we default this to `leva-documentation`?

Reply:
Added as default leva-documentation unless another one is specified